src/MainWindow.cpp

```diff
@@ -840,7 +840,8 @@ void MainWindow::doubleClickTable(const QModelIndex& index)
     editDock->allowEditing(allowEditing);
 
     // Load the current value into the edit dock
-    editDock->loadText(index.data(Qt::EditRole).toByteArray(), index.row(), index.column());
+    editDock->loadDataFromCell(index.data(Qt::EditRole).toByteArray(),
+                               index.row(), index.column());
```

Comment:
Good points.  Just updated this code to use variable names, making it short enough to fit on one line anyway. 😁